src/routes/list/containers/ListContainer.tsx

```diff
@@ -61,40 +66,148 @@ const ListContainer: React.FC<IListContainerProps> = (props): React.JSX.Element
   const [displayedCategories, setDisplayedCategories] = useState(props.categories);
   const [copy, setCopy] = useState(false);
   const [move, setMove] = useState(false);
+  const [preloadedFieldConfigurations, setPreloadedFieldConfigurations] = useState<
+    { id: string; label: string; data_type: string; position: number }[] | undefined
+  >(props.preloadedFieldConfigurations);
   const navigate = useNavigate();
 
-  // Add polling for real-time updates
+  // Prefetch field configurations once to remove first-open flicker
+  useEffect(() => {
```

Comment:
um... even if this does what we're looking for, and i think there is reasonable speculation there, this is a lot to maintain. i'm not sure its worth this